lib/reporter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var _ = require('lodash');
+var defaultFormatter = require('./defaultFormatter');
+
+module.exports = function(opts) {
+  var options = opts || {};
+
+  var formatter = options.formatter || defaultFormatter();
+
+  return function(css, result) {
+    var messagesToLog = (!options.plugins || !options.plugins.length)
+      ? result.messages
+      : result.messages.filter(function(message) {
+          if (options.plugins.indexOf(message.plugin) !== -1) {
```

Comment:
`~`?

I saw that in your selector-parser code. What is it and where is it documented? I can't seem to find anything.